docs/release_notes.rst

```diff
@@ -0,0 +1,38 @@
+
+Release Notes
+=============
+
+.. toctree::
+
+   CHANGELOG.rst
```

Comment:
Suggest putting this after the narrative.  Also, why is the .rst wrapper needed?  Why not just put the `docs/CHANGELOG.rst` lines here?

Reply:
Thanks! I could've sworn I tried that option. I added a few lines for v2.3.0 in the Narrative.

Reply:
I think it didn't work for you because of the different levels of section headers, which I updated.

Reply:
Yeah, thanks again. I was just noticing that. I have a history of issues with the section header levels.